telematic_system/telematic_apps/web_app/server/tests/nats_client/file_processing_nats_publisher.test.js

```diff
@@ -0,0 +1,25 @@
+const { CreateNatsConn } = require("../../nats_client/nats_connection");
+const {
+  pubFileProcessingReq,
+} = require("../../nats_client/file_processing_nats_publisher");
+require("dotenv").config();
+
+describe("Test NATS publisher", () => {
+  it("Test NATS publish to topic ui.file.procressing", async () => {
+    try {
+      const nc = await CreateNatsConn();
```

Comment:
We may want to establish specific naming conventions for variables as well. This article has some pretty good recommendations: https://betterprogramming.pub/useful-tips-for-naming-your-variables-8139cc8d44b5

For connections in general, maybe we always include "Conn" in the name. "nc" -> "natsConn".

Also, side question - are we using camelcase for vars? This seems inconsistent. 

@kjrush, @paulbourelly999

Reply:
Addressed https://github.com/usdot-fhwa-stol/cda-telematics/pull/174/commits/6cf46faa73ec4d2375b4bc89bf45dda79a66ad0e